bot/cogs/filtering.py

```diff
@@ -300,7 +329,7 @@ def mod_log(self) -> ModLog:
 
         Attempts to catch some of common ways to try to cheat the system.
         """
-        # Remove backslashes to prevent escape character aroundfuckery like
+        # Remove backslashes to prevent escape character around fuckery like
```

Comment:
No no. `aroundfuckery` is one word. Well, I mean, technically it isn't a word at all, but it makes zero sense as two words.
```suggestion
        # Remove backslashes to prevent escape character aroundfuckery like
```

Reply:
Replace "fuckery" with something temporarily to help you understand:

> Around something like [example]

Vs

> Aroundsomething like [example]

Reply:
The democracy decided.
![image](https://user-images.githubusercontent.com/44778521/75698240-a1646e80-5cae-11ea-98fd-098d9a14e24b.png)
`around-fuckery` it will be.

Reply:
fuckaroundery will do. even around-fuckery is okay, just please god not `around fuckery`. Anything but that.